### DISCUSSION TITLE
How to access nonDefault folders

### DISCUSSION THREAD

**Discussion by bhaskarvenna** [ecederstrand/exchangelib#962] [Q&A] [answered]
Hi, this was really helpful
Can you help me on how to access messages from nonDefault folders that is not subFolder of default folders.
I tried several ways but no luck yet.

when I used tree it looks like this root-->my main customer folder(with spaces)-->final_subFolder

Please let me know

Regards:
Bhasker.V
jsglp

**Comment by ecederstrand** [comment, marked as answer]
That should be mentioned in https://ecederstrand.github.io/exchangelib/#folders, but exchangelib supports the traversal syntax from pathlib: `account.root / 'my main customer folder' / 'final_subFolder'`

**Reply by bhaskarvenna** [reply]
@ecederstrand thanks for your inputs. that really helped. 
I'm able to reach the object(custom folders) and import those messages.

Regards:
Bhasker.V
Jsglp
